From d7986cf978e1fa66a4fdb0ef263c2e31b832d007 Mon Sep 17 00:00:00 2001
From: Wolf Lin <Wolf_Lin@pegatroncorp.com>
Date: Mon, 29 Mar 2021 16:36:52 +0800
Subject: [PATCH] Change the platform name in multiple platforms. -
 FB_6032_BN_F to FB6032_BNF - FB_6256_BN_B to FB6256_BNB - FB_6032_BN_F to
 FB6032_BNF - FN_8032_BN_F to FN8032_BNF - FN_8032_BA_F to FN8032_BAF -
 FM_6256_BN_F to FM6256_BNF - FM_6256_BA_F to FM6256_BAF

---
 miscutils/mb_detect.c | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/miscutils/mb_detect.c b/miscutils/mb_detect.c
index 16835e7..805d3dd 100644
--- a/miscutils/mb_detect.c
+++ b/miscutils/mb_detect.c
@@ -7,13 +7,13 @@
 extern void mbdetect_usage(void);
 
 SysBoardInfo sysBoardInfo[] = {
-	{ 0x04, "bugatti", 		"FB_6032_BN_F\0", "fb_6032_bn_f\0", 0, 0 },
-	{ 0x05, "jaguar",		"FB_6256_BN_B\0", "fb_6256_bn_b\0", 0, 0 },
-	{ 0x07, "bugatti2",		"FB_6032_BN_F\0", "fb_6032_bn_f\0", 1, 0 },
-	{ 0x08, "astonMartin",		"FN_8032_BN_F\0", "fn_8032_bn_f\0", 0, 0 },
-	{ 0x08, "astonMartin(BMC)",	"FN_8032_BA_F\0", "fn_8032_ba_f\0", 0, 1 },
-	{ 0x09, "gemini",		"FM_6256_BN_F\0", "fm_6256_bn_f\0", 0, 0 },
-	{ 0x09, "gemini(BMC)",		"FM_6256_BA_F\0", "fm_6256_ba_f\0", 0, 1 }
+	{ 0x04, "bugatti", 		"FB6032_BNF\0", "fb6032_bnf\0", 0, 0 },
+	{ 0x05, "jaguar",		"FB6256_BNB\0", "fb6256_bnb\0", 0, 0 },
+	{ 0x07, "bugatti2",		"FB6032_BNF\0", "fb6032_bnf\0", 1, 0 },
+	{ 0x08, "astonMartin",		"FN8032_BNF\0", "fn8032_bnf\0", 0, 0 },
+	{ 0x08, "astonMartin(BMC)",	"FN8032_BAF\0", "fn8032_baf\0", 0, 1 },
+	{ 0x09, "gemini",		"FM6256_BNF\0", "fm6256_bnf\0", 0, 0 },
+	{ 0x09, "gemini(BMC)",		"FM6256_BAF\0", "fm6256_baf\0", 0, 1 }
 };
 
 static u32 mb_id_get_from_cpld(void)
@@ -75,7 +75,6 @@ static u32 mb_id_get_from_gpio(void)
 
 	board_id = (id[3] << 3) + (id[2] << 2) + (id[1] << 1) + id[0];
 
-error:
 	return board_id;
 }
 
-- 
2.25.1

